lib/misc_test.go

```diff
@@ -137,3 +146,56 @@ func TestMisc_URL_Append(t *testing.T) {
 		test_url_append(t, tst.url1, tst.url2, tst.expected)
 	}
 }
+
+func ExampleSplitArgs() {
+	// Produces "FirstArgument" "Second Argument" "Third Argument" "Fourth Argument"
+	SplitArgs("FirstArgument \"Second Argument\" \"Third Argument\" \"Fourth Argument\"")
+
+	// Produces "FirstArgument" "Second Argument" "FourthArgument"
+	SplitArgs("FirstArgument 'Second Argument' FourthArgument")
+
+	// Produces "First" "There is a ' in the middle"
+	SplitArgs("First \"There is a ' in the middle\"")
+}
+
+/* ExampleTrackTime provides an example of TrackTime and TrackStart usage */
+func ExampleTrackTime() {
+	fmt.Printf("Example - start\n")
+
+	t1 := TrackStart()
+	for i := 0; i < 10; i++ {
+		fmt.Printf("Example - loop %d", i)
+	}
+
+	te := TrackTime(t1, "Example")
+	fmt.Printf("Example - took: %s", te)
+}
+
+func ExampleTrackTimeDeferred() {
```

Comment:
lib/misc_test.go:174: ExampleTrackTimeDeferred refers to unknown identifier: TrackTimeDeferred